src/api/api.ts

```diff
@@ -1,45 +1,49 @@
-import { toast } from 'react-toastify';
-import store from '../store/RootStore';
-import { CaretakerResponse, CaretakerSearchFilters, PagingParams } from '../types';
+import { toast } from "react-toastify";
+import store from "../store/RootStore";
+import {
+  CaretakerResponse,
+  CaretakerSearchFilters,
+  PagingParams,
+} from "../types";
+import { UserProfiles } from "../types/user.types";
 
-const backendHost = import.meta.env.VITE_BACKEND_HOST || window.location.hostname;
-const backendPort = import.meta.env.VITE_BACKEND_PORT || '8081';
+const backendHost =
+  import.meta.env.VITE_BACKEND_HOST || window.location.hostname;
+const backendPort = import.meta.env.VITE_BACKEND_PORT || "8081";
 export const PATH_PREFIX = `http://${backendHost}:${backendPort}/`;
 
-type Method = 'GET' | 'POST' | 'PUT' | 'DELETE';
+type Method = "GET" | "POST" | "PUT" | "DELETE";
 
 export type User = {
   _id: string;
   name: string;
   email: string;
-}
+};
 
 class API {
-
   async fetch<T>(
     method: Method,
     path: string,
     body?: unknown,
-    headers: HeadersInit = {},
+    headers: HeadersInit = {}
   ): Promise<T> {
-    
     const options = {
       method,
       headers: {
-        'Content-Type': 'application/json',
-        'X-XSRF-TOKEN': store.user.xsrfToken,
-        ...headers
+        "Content-Type": "application/json",
```

Comment:
Dobrze, że ujednoliciłeś te cudzysłowy, niech zawsze będą podwójne zamiast pojedynczych. Dodam to do jako regułę do eslinta.